### ISSUE TITLE
Readme for KV

### ISSUE THREAD

**Issue by victoriabernard92** [cloudflare/wrangler#510]
#### Overview and problem statement

Add to readme

#### Basic example

* A copy pasteable .json file example for bulk uploads instead of just the definition
* Example of how to set up a file to be the value of a key (that is possible with the commands right 🤔?)
* what `--prefix` does

**Comment by ashleymichal** [comment]
> A copy pasteable .json file example for bulk uploads instead of just the definition

I don't think this would make a whole lot of sense, as the json file would include a lot of custom keys defined by the user

> Example of how to set up a file to be the value of a key (that is possible with the commands right 🤔?)

This will be supported by a set of `kv:bucket` subcommands in 1.4.0 (upload, delete, sync)

> what --prefix does

Good call!

**Comment by victoriabernard92** [comment]
Thanks! Though yes a program will likely be generating the file, it does make it harder to understand what to make that program generate without a clear example (e.g. you sharing that example was used to help me understand so why not share that with users in the docs? )

**Comment by gabbifish** [comment]
The readme is also a little outdated, now that I read it...
for example, 
```sh
$ wrangler kv:key list f7b02e7fc70443149ac906dd81ec1791 --prefix="public"
🌀  Retrieving keys 🌀 
✨  Success:
+------------------+----------------------------------+
| KEY              | EXPIRATION                       |
+------------------+----------------------------------+
| "key"            | Wed Aug 28 10:28:44 CDT 2019     |
+------------------+----------------------------------+
```
is inaccurate--we return a json blob instead.
It could also be worth adding the whole invocation (what input looks like and what output looks like) for the `kv:key` and `kv:bulk` operations.

**Comment by gabbifish** [comment]
Closed with #537
